### PR TITLE
Fix multiple typos [2]

### PR DESCRIPTION
**Pull request by JonHanna**

### CODE REVIEW

**Comment by stephentoub** [comment]
@JonHanna, your 2/3/4 sets of changes all have merge conflicts.  Can you rebase?

**Comment by JonHanna** [comment]
Yeah, on it.

(Obviously I've been trying to batch these to be reasonably reviewable chunks that don't conflict with each other, but two conflicts got through).

**Comment by JonHanna** [comment]
Test Innerloop Windows_NT Debug Build and Test

**Comment by stephentoub** [comment]
@JonHanna, what are the failures that are causing you to re-test?  Are there issues opened for them?

**Comment by JonHanna** [comment]
This one was a failure on System.Tests.EnvironmentTickCount.TickCountTest which might be genuine, though it seems fine locally. Most of the others have been timeouts.

**Comment by stephentoub** [comment]
> This one was a failure on System.Tests.EnvironmentTickCount.TickCountTest which might be genuine, though it seems fine locally. Most of the others have been timeouts.

Ok, thanks.  In the future, please check to see whether there's already an issue opened: if there is, please ref count it by adding the failed leg link, and if there isn't, please open one.  We really want to drive these kinds of spurious failures to 0.

File: src/System.Data.SqlClient/src/Microsoft/SqlServer/Server/SmiMetaData.cs [hunk 1]
```diff
@@ -139,7 +139,7 @@ internal static SmiMetaData DefaultNVarChar
         }
 
 
-        // SMI V100 (aka V3) constructor.  Superceded in V200.
```
Comment (stephentoub) [inline src/System.Data.SqlClient/src/Microsoft/SqlServer/Server/SmiMetaData.cs]: FWIW, "supercede" is generally considered to be a valid alternate spelling.

Comment (JonHanna) [inline src/System.Data.SqlClient/src/Microsoft/SqlServer/Server/SmiMetaData.cs]: For the most part I've left variants alone, but supercede is relatively rare and sometimes considered incorrect. While I don't consider it so, I decided to change it:

Citations for supercede and supersede

Comment (JonHanna) [inline src/System.Data.SqlClient/src/Microsoft/SqlServer/Server/SmiMetaData.cs]: On a similar note, the use of periods with abbreviations is mostly a matter of style, and I personally prefer to leave them out, but with the older Latin-based abbreviations using them is more common that not using them so I've been changing to that when I've come across "ie", "eg", "etc", etc.

**Comment by stephentoub** [comment]
LGTM

**Pull request merged**